desktop/apps/buy_art/templates/landing.jade

```diff
@@ -7,14 +7,14 @@ include meta
         .buy-art-header__title.buy-art-text-center-on-mobile= header.headline
         .buy-art-show-on-desktop
           .buy-art-header__signup-buttons
-            a.ba__email-signup-button( href='emaillink' )
+            a.ba__email-signup-button( href='/signup' )
               img.buy-art__svg-icon(src=header.email_icon)
               | Sign up using email
-            a.ba__facebook-signup-button( href='fblink' )
+            a.ba__facebook-signup-button( href='/users/auth/facebook' )
```

Comment:
You can also add the redirect url here too if needed (`?redirect-to=/personalize`).